mantidproject/mantidimaging
# Removes `pg_image_view` and replaces it with `mi_image_view`

DON'T MERGE BEFORE #704 as I was lazy and it's based on that.

Removes `pg_image_view` and replaces it with `mi_image_view` which has had some refactoring (into a VP) done and has some tests around the ROI. Functionally it should be exactly the same.

To test:
- Check that < and > buttons still work
- Check that clicking the timeline goes to the correct place

Places that the MIImageView is used:
- Compare dialog / stack choice - check that the same things from "To test" are still working there
- Select ROI on ROI Norm and Crop


Fixes #704
